Source1.cpp

```diff
@@ -39,6 +39,7 @@ bool check_repeat(int* tempPuzzle)
 int* deleteBlanks(int steps[], int& stepCount)
 {
     int* stepsSimplified = new int[stepCount];
+    memset((char*)stepsSimplified, 0xcd, sizeof(int)*stepCount);
```

Comment:
In https://github.com/DancingNeko/Tile-Puzzle/blob/0695b5988a20a148f889dd20e8fd3aebe4418d1b/Source1.cpp#L44, we need to quit loop by checking its value. 
So, here we have to initialize the initial value of this chunk of memory to a invalid value (0xcd).